ElectronicObserverTypes/Extensions/ShipDataExtensions.cs

```diff
@@ -470,15 +473,13 @@ ShipTypes.SeaplaneTender or
 
 		{ MasterShip.ShipType: ShipTypes.LightAircraftCarrier } => ship.ASWTotal switch
 		{
-			>= 100 => ship.HasSonar() && ship.HasAntiSubmarineAircraft(),
-			>= 65 => ship.HasSpecialAntiSubmarineAttacker() || ship.HasAswPatrolAircraft(),
+			>= 100 => ship.AswCondition100() || ship.AswCondition65() || ship.ZeroSlotASWCond(),
```

Comment:
Shouldn't this be
```cs
(...) && ship.ZeroSlotASWCond()
```
?

Though I'd put the 0 slot condition in the 100 and 65 condition functions, since that's part of those conditions anyway, no?